terraform/modules/data-share-service/outputs.tf

```diff
@@ -16,3 +16,6 @@ output "consumer_client_id" {
 output "consumer_client_secret" {
   value = module.cognito.consumer_client_secret
 }
+output "alarm_action" {
```

Comment:
why is this being outputted?

Reply:
oh yep shouldn't be - this was how I originally did it